tools/libboard/board_d5005/CMakeLists.txt

```diff
@@ -32,3 +32,8 @@ opae_add_module_library(TARGET board_d5005
         board_common
     COMPONENT opaeboardlib
 )
+
+target_include_directories(board_d5005
+    PRIVATE
+        ${OPAE_SDK_SOURCE}/ibboard/board_common
```

Comment:
s,ibboard,libboard,
Is this needed?  Seems to be building without it.

Reply:
fixed libboard path